### ISSUE TITLE
DiC for inpaiting Task Problem

### ISSUE THREAD

**Issue by ALable** [YuchuanTian/DiC#9]
I try to use DiC model to inpainting eye. My input is a face picture with eye mask and I expect to generate full face but maybe meet some problems

left is groud truth and right is generate through diffusion.ddim_sample_loop method
<img width="814" height="708" alt="Image" src="https://github.com/user-attachments/assets/5a4cdb7a-73f8-443e-95a6-e210388aeea0" />

Here are my core codes following。

> x = vae.encode(image * (1.0 - mask.unsqueeze(1))).latent_dist.sample().mul_(0.18215)
> gen_latent = diffusion.ddim_sample_loop(
>                    model,
>                    shape=x[:n_vis].shape, 
>                    noise=None,
>                    model_kwargs={k: v[:n_vis] for k, v in model_kwargs.items()},
>                    device=device,
>                     progress=False
>                                 )
> x_gen = vae.decode(gen_latent / 0.18215).sample.float().detach().cpu()

I want to know why the corresponding subject information image cannot be generated.

**Comment by ALable** [comment]
I had motified my code about noise input to fit in inpatining/image-to-image task , but I'm still skeptical about this model's inpainting ability

**Comment by YuchuanTian** [comment]
Thanks for your trial on the inpainting task! Applying DiC to inpainting is interesting. I am curious how did you apply DiC to inpainting? Thank you!
